lang/python/host-pip-requirements/lib2to3.txt

```diff
@@ -0,0 +1,2 @@
+2to3==1.0 --hash=sha256:958bc212c928bbdcbc778b72528e0a39ae4ee8040eda6af6c3b5dd640c98ce6d
```

Comment:
I would prefer to do something similar to Debian/Ubuntu (their `2to3` comes from a simple [script](https://salsa.debian.org/cpython-team/python3-defaults/-/blob/e429bbce303776b0357ef2636fd4b43ee0945c57/debian/2to3.py) that they maintain) rather than install a random `2to3` from PyPI.

I'll try to find time in the coming week to add a 2to3 host package, then this package can do `HOST_BUILD_DEPENDS:=2to3/host`.

Reply:
> …why do you say « random » ?

* Project doesn't have a readme or a public code repository, only has one maintainer and one release from 2018
* Maintainer only has 2 projects total, the other one is labelled as deprecated

Reply:
Great... Thanks !

Reply:
> If needed, I can also take a look, test, or make the proposed PR, but I don’t know when.

I try to make the package for 2to3/host...
Will propose a PR if I get success on it !

Reply:
Package made and compiled on master...
Waiting for your advice and review.
;-)

Reply:
https://github.com/openwrt/packages/pull/17429